src/test/java/fi/helsinki/cs/tmc/core/commands/DownloadModelSolutionTest.java

```diff
@@ -5,10 +5,14 @@
 import fi.helsinki.cs.tmc.core.communication.TmcApi;
 import fi.helsinki.cs.tmc.core.domain.Course;
 import fi.helsinki.cs.tmc.core.domain.Exercise;
+import java.io.IOException;
+
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.anyString;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.mock;
```

Comment:
Import order should be:
- All static imports in a single group
- fi.helsinki.cs.tmc imports (only if this source file is in the fi.helsinki.cs.tmc package space)
- Third-party imports, one group per top-level package, in ASCII sort order
- for example: android, com, junit, org, sun
- java imports
- javax imports